api/helpers/errors.js

```diff
@@ -44,10 +46,87 @@ class ILSIntegrationError extends Error {
   }
 }
 
+class PatronNotFound extends Error {
+  constructor() {
+    super();
+    this.type = "patron-not-found";
+    this.name = "PatronNotFound";
+    this.message = "The patron couldn't be found.";
```

Comment:
Can this message be more user-friendly? It is kind of confusing when displayed on the client.

Reply:
Sure. I'm going to update it to `"The patron couldn't be found in the ILS with the barcode or username."` since this happens when requesting a patron from the ILS.